example/src/main/java/org/apache/rocketmq/example/tracemessage/TraceProducer.java

```diff
@@ -27,6 +27,7 @@
     public static void main(String[] args) throws MQClientException, InterruptedException {
 
         DefaultMQProducer producer = new DefaultMQProducer("ProducerGroupName",true);
+        producer.setNamesrvAddr("red-rocketmq-namesrv.sit.devops.xiaohongshu.com:9876");
```

Comment:
Here,please modify your code!Here,it shouldn't come out yourself enviroment info!

Reply:
Sorry for the inconvenience.  Committed but forgot to push it. Correct it now.